### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by modelonrobinandersson**
This closes https://github.com/modelon-community/PyFMI/pull/27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,20 @@ For citing PyFMI, please use:
 Andersson, C, Åkesson, J & Führer, C 2016, PyFMI: A Python Package for Simulation of Coupled Dynamic Models with the Functional Mock-up Interface. Technical Report in Mathematical Sciences, nr. 2, vol. 2016, vol. LUTFNA-5008-2016, Centre for Mathematical Sciences, Lund University.
 
 For information about contributing, see [here](https://github.com/modelon/contributing).
+
+Installation from source
+-----------
+PyFMI can be built and installed from source, note this requires [FMI library](https://github.com/modelon-community/fmi-library). Additional details are found in the file `setup.py`.
```
Comment (PeterMeisrimelModelon) [inline README.md:21]: also Assimulo and dependencies within?

Maybe worthwhile to update Assimulo Readme in the same turn.

File: doc/sphinx/source/installation.rst [hunk 2]
```diff
@@ -4,31 +4,4 @@
 Installation
```
Comment (PeterMeisrimelModelon) [inline doc/sphinx/source/installation.rst:4]: Might as well remove this directly?

**Review by PeterMeisrimelModelon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (modelonrobinandersson) [reply README.md:21]: Will do!

File: doc/sphinx/source/installation.rst (hunk 2 above)
Comment (modelonrobinandersson) [reply doc/sphinx/source/installation.rst:4]: I think yes but it will probably have to happen within a future release, unaware of this used anywhere.

**Pull request merged**